tests/test_orient_sync_voting.py

```diff
@@ -119,6 +112,41 @@ def test_estimate_shifts(source_orientation_objs):
     assert np.allclose(est_shifts, src.offsets)
 
 
+def test_estimate_rotations_fuzzy_mask():
+    noisy_src = Simulation(
+        n=35,
+        vols=AsymmetricVolume(L=128, C=1, K=400, seed=0).generate(),
+        offsets=0,
+        amplitudes=1,
+        noise_adder=WhiteNoiseAdder.from_snr(snr=2),
+        seed=0,
+    )
+
+    # Orientation estimation without fuzzy_mask.
+    max_shift = 1 / noisy_src.L
+    shift_step = 1
+    orient_est = CLSyncVoting(
+        noisy_src, max_shift=max_shift, shift_step=shift_step, mask=False
+    )
+    orient_est.estimate_rotations()
+
+    # Orientation estimation with fuzzy mask.
+    orient_est_fuzzy = CLSyncVoting(
+        noisy_src, max_shift=max_shift, shift_step=shift_step
+    )
+    orient_est_fuzzy.estimate_rotations()
+
+    # Check that fuzzy_mask improves orientation estimation.
+    mean_angle_dist = mean_aligned_angular_distance(
+        orient_est.rotations, noisy_src.rotations
+    )
+    mean_angle_dist_fuzzy = mean_aligned_angular_distance(
+        orient_est_fuzzy.rotations, noisy_src.rotations
+    )
+
+    assert mean_angle_dist_fuzzy < mean_angle_dist < 10
```

Comment:
This makes me a little nervous. Are we sure it does what we think it does?

Reply:
In this case I think it does what's expected, but probably could have unexpected behavior in certain cases. Replaced this with two separate asserts. One to check that the estimates are reasonable. Another to check that `fuzzy_mask` gives us an improvment.

Also, switched over to using `np.testing.assert_*` while I was in there.